data/data/agent/files/usr/local/bin/add-node.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# shellcheck disable=SC1091
+source issue_status.sh
+
+BASE_URL="${SERVICE_BASE_URL}api/assisted-install/v2"
+
+cluster_id=""
+while [[ "${cluster_id}" = "" ]]
+do
+    # Get cluster id
+    cluster_id=$(curl -s -S "${BASE_URL}/clusters" | jq -r .[].id)
+    if [[ "${cluster_id}" = "" ]]; then
+        sleep 2
+    fi
+done
```

Comment:
Do we need a timeout followed by error message here?

Reply:
Good question. I think here, the cluster_id should be available fairly quickly as the agent-add-node service has an After dependency on apply-host-config.service, which itself, has a dependency on agent-register-infra.service and it on agent-import-cluster.service. So by the time this script/service runs the cluster_id should be available.

@sadasu What is a status_issue? Could you explain? Thanks!

Reply:
Yeah, as per the [agent services architecture](https://github.com/openshift/installer/blob/master/docs/user/agent/agent-services.md) I'd expect that in case of issue the `agent-register-cluster` would prevent the other services, the loop here it's just to cope in case of slower execution (similar approach in another scritps)

Reply:
@sadasu Please disregard my question. I see status_issue updated below.